src/Kafka/Settings/Settings.tsx

```diff
@@ -0,0 +1,164 @@
+import {
+  Button,
+  Card,
+  CardBody,
+  Flex,
+  FlexItem,
+  Modal,
+  ModalVariant,
+  Page,
+  PageSection,
+  Spinner,
+  Switch,
+} from "@patternfly/react-core";
+import type { FunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import { SettingsAlert } from "./components";
+import type { AlertStatus, SettingsStatus } from "./types";
+
+export type SettingsProps = {
+  connectionStatus: SettingsStatus;
+  onSwitchClick: () => void;
+  isModalOpen: boolean;
+  onClickTurnOff: () => void;
+  alertStatus: AlertStatus;
+  connectionState: boolean;
+  onClickClose: () => void;
+};
+
+export const Settings: FunctionComponent<SettingsProps> = ({
+  connectionStatus,
+  onSwitchClick,
+  isModalOpen,
+  onClickTurnOff,
+  alertStatus,
+  onClickClose,
+  connectionState,
+}) => {
+  const { t } = useTranslation("kafka");
+
+  return (
+    <>
+      <PageSection>
+        <Card>
+          <CardBody>{t("settings.kafka_instance_settings_label")}</CardBody>
+        </Card>
+      </PageSection>
+      <Page>
+        <PageSection>
+          <Card>
```

Comment:
This could be a follow-up PR, but the original designs for this page set a max-width for the card. 

The only support PF provides for setting a max-width is on the Form component, which you can see in [this example](https://www.patternfly.org/v4/components/form#limit-width). I think there are two options for applying this style, and I'm not sure which would be the preferred method:

1. Wrap the card in a Form component with the limit width prop. 
2. Create custom CSS that applies a similar style to the card: `max-width: 55rem`